src/applications/claims-status/utils/helpers.js

```diff
@@ -43,6 +43,43 @@ export function getStatusDescription(status) {
   return statusStepMap[status];
 }
 
+const claimPhaseTypeStepMap = {
+  CLAIM_RECEIVED: 'Step 1 of 8: Claim received',
+  UNDER_REVIEW: 'Step 2 of 8: Initial review',
+  GATHERING_OF_EVIDENCE: 'Step 3 of 8: Evidence gathering',
+  REVIEW_OF_EVIDENCE: 'Step 4 of 8: Evidence review',
+  PREPARATION_FOR_DECISION: 'Step 5 of 8: Rating',
+  PENDING_DECISION_APPROVAL: 'Step 6 of 8: Preparing decision letter',
+  PREPARATION_FOR_NOTIFICATION: 'Step 7 of 8: Final review',
+  COMPLETE: 'Step 8 of 8: Claim decided',
+};
+
+export function getClaimPhaseTypeHeaderText(claimPhaseType) {
+  return claimPhaseTypeStepMap[claimPhaseType];
+}
+
+const claimPhaseTypeDescriptionMap = {
+  CLAIM_RECEIVED: 'We received your claim in our system.',
+  UNDER_REVIEW:
+    'We’re checking your claim for basic information, like your name and Social Security number. If information is missing, we’ll contact you.',
+  GATHERING_OF_EVIDENCE:
+    'We’re reviewing your claim to make sure we have all the evidence and information we need. If we need anything else, we’ll contact you.',
+  REVIEW_OF_EVIDENCE:
+    'We’re reviewing all the evidence for your claim. If we need more evidence or you submit more evidence, your claim will go back to Step 3: Evidence gathering.',
+  PREPARATION_FOR_DECISION:
+    'We’re deciding your claim and determining your disability rating. If we need more evidence or you submit more evidence, your claim will go back to Step 3: Evidence gathering.',
+  PENDING_DECISION_APPROVAL:
+    'We’re preparing your decision letter. If we need more evidence or you submit more evidence, your claim will go back to Step 3: Evidence gathering.',
+  PREPARATION_FOR_NOTIFICATION:
+    'A senior reviewer is doing a final review of your claim and the decision letter. If we need more evidence or you submit more evidence, your claim will go back to Step 3: Evidence gathering.',
```

Comment:
This isn't so much feedback on the pull request as it is a question re: the design. The language here is for step 7, and says that you can go back to Step 3 potentially. However, in the designs I've seen, the diagram shows that you can only go back to Step 3 between steps 4 - 6, not 7. @pmclaren19 @uxgary 

![image](https://github.com/department-of-veterans-affairs/vets-website/assets/105806934/2d783d5d-4bdc-484a-9c5e-e6c81d777310)

Reply:
https://dsva.slack.com/archives/C056854SVT8/p1717168745828499?thread_ts=1717166351.850769&cid=C056854SVT8 @samcoforma Sounds like after somme workshops we learned the claim can revert to step 3 at any step. @uxgary is going to create a new ticket to update the diagram but we can leave this language as is!